x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/persistence_modification_sublime_app_plugin_or_script.json

```diff
@@ -0,0 +1,50 @@
+{
+  "author": [
+    "Elastic"
+  ],
+  "description": "Adversaries may create or modify the Sublime application plugins or scripts to execute a malicious payload each time the Sublime application is started.",
+  "false_positives": [
+    "Legitimate Sublime plugins or application updates"
```

Comment:
same rule of thumb here: this isn't useful information without giving examples of what is and is not legitimate

Reply:
Application updates is quite self-explanatory, most known FPs are excluded:

![image](https://user-images.githubusercontent.com/64742097/107642543-518b7700-6c75-11eb-8764-35cbba31e6d2.png)